src/boot/stylesheets/actions.scss

```diff
@@ -57,7 +57,7 @@
 	}
 }
 
-.wpnc__action-link {
+.wpnc__main .wpnc__action-link {
```

Comment:
> The general resets do need to go under .wpnc__main otherwise Calypso will pick up those rules as well

Of course! I knew it felt funny for a reason 🙃 

I think it _did_ work, but then I saw "Avoid using over-qualified selectors like div.my-class" in the [Sass coding guidelines](https://wpcalypso.wordpress.com/devdocs/docs/coding-guidelines/css.md). I admit I haven't totally grokked those guidelines as a whole (and have never worked with Sass before 😱), so I just went with this.